sass/blocks/article_offer.sass

```diff
@@ -0,0 +1,149 @@
+// Article offers
+
+.article__offer--headingPersonal
+  background-color: $color__green
+  position: relative
+  display: block
+  height: 22rem
+  padding-top: 3rem
+
+.article__offer--headingPersonal h3
```

Comment:
I see *why* you did this, but in the future it's _best_ to just add a class to the element you'd like to style, and not look for a child based off of an element selector.